toolkit/tools/imager/imager.go

```diff
@@ -37,23 +32,23 @@ var (
 	logLevel        = exe.LogLevelFlag(app)
 )
 
-const (
-	// additionalFilesTempDirectory is the location where installutils expects to pick up any additional files
-	// to add to the install directory
-	additionalFilesTempDirectory = "/tmp/additionalfiles"
-
-	// postInstallScriptTempDirectory is the directory where installutils expects to pick up any post install scripts
-	// to run inside the install directory environment
-	postInstallScriptTempDirectory = "/tmp/postinstall"
-
-	// sshPubKeysTempDirectory is the directory where installutils expects to pick up ssh public key files to add into
-	// the install directory
-	sshPubKeysTempDirectory = "/tmp/sshpubkeys"
+func populateImagerConfig() *imager.Config {
+	const defaultSystemConfig = 0
```

Comment:
Looks like a duplicate from `main` and I think you're also not using the `SystemConfig` field below.